playwright_test/locaterPackage/kmLocators.js

```diff
@@ -40,3 +40,7 @@ export const richMessagesLocators = {
     kmSubmitButtonResponse: '//div[text()="Button Response"]',
     kmImageResponse: '//div[@class="km-image-template"]'
 };
+
+export const commonValue = {
```

Comment:
make it COMMON_VALUES

Reply:
can't start with capital letters ill make that common_Values

Reply:
done

Reply:
why not?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

Reply:
I mean it is not recommended to start with a capital letter if I convert this into capital then I have to convert all the names to capital.
If you say ill do that I don't have any issue

Reply:
"not recommended" 
>> where did you read this? as far as i understand, const are usually named in all CAPS so that it is easy to understand for other developers that this is a constant, and not a variable

Reply:
I have gone through a lot of GitHub codes where people are using this method only but it fil ill do the change 😊